debezium-server/pom.xml

```diff
@@ -109,6 +109,21 @@
             <groupId>io.apicurio</groupId>
             <artifactId>apicurio-registry-utils-converter</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@jeremy-l-ford Why are the dependencies necessary to be explcitily defined? Also if they are in the test deps block should not they be marked as test scope?

Reply:
@jpechane - Locally I was seeing 2.3.1 being pulled in as a transitive for these jars which continued to cause issues.    Since apicurio-registry-utils-converter is not marked as a test scoped dependency, I thought that these additional jars should also not be test scoped.

Reply:
@jeremy-l-ford Thanks for the explanation. I ned to check if this is an error in scope or just a position in pom.
I am not going to merge this PR until https://issues.redhat.com/browse/DBZ-5977 is resolved. In that case the changes here should be only relevant to the issue and not handle anything else.

Reply:
@jpechane From my reading of [DBZ-5282](https://issues.redhat.com/browse/DBZ-5282), it boils down to just needing the apicurio dependencies updated.

Reply:
@jeremy-l-ford Apicuri dependencies has been updated and the new nightly image was published. Could you take a look if it is possible to write a test for the SSL setup?